conceptual/deployment/distributing.md

```diff
@@ -15,12 +17,12 @@ Your project may _flow_ the necessity to use the aspect framework to consuming p
 * Your project has public classes that have _reference validators_.
 * Your project contains a _transitive project fabric_ or references a project that contains one.
 
-If this is the case of your project, you do not need to take any action. Your package reference to `Metalama.Framework` will flow to the consumers of your project.
-
+If this is the case in your project, you do not need to take any action. Your package reference to `Metalama.Framework` will flow to the consumers of your project.
 
+<!-- then this becomes  ##Preventing the transmission of the use of Aspects  -->
 ## Avoiding to flow the use of aspects
 
-If, conversely, the consumers of your project will _not_ need to use aspects just because of your project, you can prevent the `Metalama.Framework` to flow to the consumers of your project by setting the `PrivateAssets="all"` property to the package reference. Additionally, you need to include the `Metalama.Framework.Redist` package, which is the only package that needs to flow to consumers.
+If, conversely, the consumers of your project will _not_ need to use aspects just because of your project, you can prevent the `Metalama.Framework` <!--- and this becomes from transmitting aspects to --> to flow to the consumers of your project by setting the `PrivateAssets="all"` property to the package reference. Additionally, you need to include the `Metalama.Framework.Redist` package, which is the only package that needs to flow to consumers.
```

Comment:
```suggestion
If, conversely, the consumers of your project will _not_ need to use aspects just because of your project, you can prevent the `Metalama.Framework` from flowing to the consumers of your project by setting the `PrivateAssets="all"` property to the package reference. Additionally, you need to include the `Metalama.Framework.Redist` package, which is the only package that needs to flow to consumers.
```